application/cms/forms.py

```diff
@@ -184,20 +180,16 @@ def __call__(self, form: "MeasureVersionForm", field):
         label="Suppression rules and disclosure control (optional)",
         hint="If any data has been excluded from the analysis, explain why",
         extended_hint="_suppression_and_disclosure.html",
-        strip_whitespace=True,
```

Comment:
"Interesting" that we had this on a checkbox field to start with.

Reply:
That's just github doing some weird code collapsing. It's on a textarea field here.